base/uk.ac.stfc.isis.ibex.activemq/src/uk/ac/stfc/isis/ibex/activemq/message/MessageParser.java

```diff
@@ -85,6 +80,7 @@ public void setActiveMQConsumer(MessageConsumer consumer) {
      *             Thrown if close fails.
      */
     public void closeJMSConsumer() throws JMSException {
+        setRunning(false);
         jmsConsumer.close();
         
```

Comment:
If we close () when it is receiving are we OK i.e. it throws an exception that we catch?